vcluster/configure/vcluster-yaml/external/platform/api-key.mdx

```diff
@@ -2,28 +2,262 @@
+        createRBAC: [[VAR:CREATE RBAC:true]]
+  `}
+      language="yaml"
+        />       
+      </Step>
+    </Flow>
+  </TabItem>
+  
+  <TabItem value="individual" label="Individual user access">
+
+  ### Connect a virtual cluster to the platform with an individual user access key
+
+  Authenticate a virtual cluster with the platform using a personal access key.
+
+  ### Prerequisites
+
+  Before you begin, ensure you have: 
+
+  - Access to the vCluster Platform.
+  - The [vCluster CLI installed](/platform/install/quick-start-guide) installed on your local machine.
+  - The ability for each user to generate their own access key from the platform console.
+    
+  ### Connect a virtual cluster to the platform using individual user access
+
+  <Flow>
+    <Step title="Locate or create the vcluster.yaml file">
+
+    You must configure `vcluster.yaml` to define how your virtual cluster connects to the platform.
+
+    - If you are creating a new virtual cluster, create a new `vcluster.yaml` file in your working directory.
+    - If you are modifying an existing virtual cluster, locate the `vcluster.yaml` file used during its deployment.
+
+    :::note
+    The `vcluster.yaml` file is typically stored locally in your project folder or provided as part of the Helm configuration used to deploy the virtual cluster.
+    :::
+
+    </Step>
+
+    <Step title="Configure and deploy vcluster.yaml for individual user access">
+
+    Update your `vcluster.yaml` file to enable individual user authentication:
+
+    ```yaml title="vcluster.yaml"
+    external:
+      platform:
+        userBased: true
+    ```
+
+    :::note
+    This configuration requires each user to authenticate with their personal access key instead of creating a shared Kubernetes Secret.
+    :::
+
+    Then in the vCluster CLI, [deploy the updated configuration](/vcluster/manage/deploy-changes/):
+
+<InterpolatedCodeBlock
+  code={`vcluster create --upgrade [[VAR:VCLUSTER NAME:VCLUSTER_NAME]] -n [[VAR:NAMESPACE:VCLUSTER_NAMESPACE]] -f vcluster.yaml`}
+  language="bash"
+  title="Upgrade vCluster using vcluster.yaml"
+/>
+    </Step>
+
+    <Step title="Generate a personal access key">
+
+    To connect to the virtual cluster using individual access, each user must:
+
+      1. Log in to the vCluster Platform console.
+      2. Navigate to **Users** and click **Access Keys**.
+      3. Click **Create Access Key**, configure the settings by providing a display name and optionally limiting its scope to specific clusters, namespaces, or virtual clusters, and save to generate a new personal access key.
+      4. Copy and securely store the access key.
+
+      :::note
+      To view your generated access keys, navigate to **Users** > **Access Keys**.
+      :::
+
+    </Step>
+
+    <Step title="Connect to the virtual cluster">
+
+    Each user can connect to the virtual cluster by using their personal access key through the vCluster CLI.
+
+    Log in to the platform:
```

Comment:
I don't think we need to be repeating how to log in here.

Reply:
Oh, I see now, we are explaining how to log in with access key. 
I think we are mixing two different things:
a) guide how to connect a vcluster with access key token
b) guide how to use access key as an individual